### ISSUE TITLE
Tab Stash opens on startup

### ISSUE THREAD

**Issue by Fluqz** [josh-berry/tab-stash#366]
Hey Guys,

Tab Stash opens up on every new Browser Tab that is opened in a new window. How can I disable Tab Stash from opening on Browser startup and on new windows? 

### System Details

- Macos Ventura 13.2
- Firefox 113.0.1
- Tab Stash 2.12.0.1

**Comment by josh-berry** [comment]
Hi, Firefox will open the sidebar based on whether it was opened in another window or not—so probably there is some other window with Tab Stash open in a sidebar, which is causing Firefox to think you want it to be open in the new window you're creating.

This behavior is unfortunately controlled entirely by Firefox—wish I could offer you a better answer, here. :/ Hope this helps though!

**Comment by Fluqz** [comment]
Hey, thanks for your reply. I have two seperate profiles/users on my macos for work and private. On one the sidebar opens whenever a new window is opened. On the other its not! I think i remember setting a setting for it to work, but i cant seem to find it.

**Comment by SudNeela** [comment]
Sorry for the bump on a closed issue, but I've been having this issue too. I also remember a setting (in the tab stash settings) - something along the lines of "tab stash behavior on browser startup", but I may be misremembering.

However, as a temporary solution, I've found that toggling the `browser.engagement.sidebar-button.has-used` (in `about:config`) to false has made tab stash not open on starting up. As josh-berry mentions above, "This behavior is unfortunately controlled entirely by Firefox".

I've found out that clicking on the tab stash button or toggling it with keyboard shortcuts doesn't set this value to true. I don't know how it was set to be true as pressing on the "show sidebars" button sets it to true. Pretty weird.

Disclaimer: I have no idea if the `browser.engagement.sidebar-button.has-used` is the thing that's controlling this behavior. From the sound of it, it _does_ seem like it, but considering that I have no clue where to find the documentation for Firefox's configs, I don't know what it truly does. I've simply observed that pressing the "show sidebars" button sets it to true.
